LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-Copyright 2022 Hedera Hashgraph LLC
+Copyright 2022 Hiero Hashgraph LLC
```

Comment:
This is not correct and must be fixed

Reply:
I put `Copyright 2024 Hiero`

Reply:
@nathanklick looks like the file header was fixed https://github.com/hiero-ledger/tsc/issues/64. Could we get an approval on this?